invisible_cities/calib/calib_functions_test.py

```diff
@@ -92,3 +92,26 @@ def test_filter_limits_outside():
     assert len(filtered_dlimits) < len(unfiltered_dlimits)
     assert len(filtered_llimits) % 2 == 0
     assert len(filtered_dlimits) % 2 == 0
+
```

Comment:
I would like this commit ("Protect changes") and "Added tests for changes" to be just one commit. You can squash the latter into the former. Ask me if in doubt.